### ISSUE TITLE
IWT doesn't work with tf 2.5

### ISSUE THREAD

**Issue by ngcthuong** [zaccharieramzi/tf-mwcnn#2]
I faced this error while trying to use the IWT layer in TF 2.5.   
![image](https://user-images.githubusercontent.com/6524339/126060381-cbd3dce1-07a2-4ff0-9b72-d5b09c0e67f6.png)
Do you have any suggestions to resolve this issue? Thanks.

**Comment by zaccharieramzi** [comment]
Hi @ngcthuong ,

Thank you for reporting this, I will look into it !

**Comment by ngcthuong** [comment]
A few line of codes in pyTorch turns out to be very troublesome in Tensorflow 😅. Thank you.

**Comment by zaccharieramzi** [comment]
Yes exactly, this has been known for quite some time now that tensor slice assignment is a disaster in TensorFlow, yet there does not seem to be any movement in the direction of fixing it.
I will try the package I created for it `tf-slice-assign`.

**Comment by ngcthuong** [comment]
Thank you for the update. I hear that Google themself moving away from TF. Anyway, thanks you and good luck with your research.

**Comment by zaccharieramzi** [comment]
Hi @ngcthuong ,

I didn't manage to reproduce the bug you reported (for TF versions 2.2, 2.4, 2.5).
Could you:
- give me a snippet of a minimal reproducible example?
- copy-paste (not with a screenshot) the entire stacktrace of the bug?

Thanks in advance

**Comment by zaccharieramzi** [comment]
Hi @ngcthuong ,

Did you manage to solve this issue?
